speech/explore-api/index.html

```diff
@@ -0,0 +1,36 @@
+<!--
+/*
+ *
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<html>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script src="js/app.js"></script>
+  <script src="key.js"></script>
+<body>
+  <div id="post-load-div" style="display:none">
+    <!-- Hidden until gapi loads -->
+    <p>
+      <input type="file" id="inputFile"></input>
+      <button onClick="handleFile()">Transcribe</button>
+    </p>
+    <p>
+      <textarea id="results" style="width:500px; height:200px;"></textarea>
```

Comment:
I recommend changing this to a `div`, removing the width and height limits, and prepending the results to this div (each as their own div).